src/index.js

```diff
@@ -1 +1,40 @@
 import './style.css';
+import addScore from './modules/addScore.js';
+import createId from './modules/createId.js';
+import refresh from './modules/refresh.js';
+
+const formStart = document.getElementById('start-form');
+const form = document.getElementById('form');
+const refreshBtn = document.getElementById('btn-refresh');
+
+document.addEventListener('DOMContentLoaded', () => {
+  if (formStart) {
+    formStart.addEventListener('submit', (e) => {
+      e.preventDefault();
+      let gameId = createId();
+      gameId.then((response) => {
+        gameId = response.result;
+        gameId = gameId.substr(14, 20);
+
+        // store game ID into local storage
+        if (gameId) localStorage.setItem('gameId', gameId);
+        formStart.reset();
+      });
+    });
+  }
```

Comment:
- [ ] @ismayito Great work on adding API to store the game scores, however when I added a new game and scores I got the console errors below and I couldn't see any game scores, kindly consider fixing the error. 
![image](https://github.com/ismayito/Leaderboard-list-app/assets/61408860/29b4d67a-5cee-4efb-883f-70378e64c78f)